sparse/tests/test_core.py

```diff
@@ -136,6 +137,10 @@ def test_dot():
     assert_eq(a.dot(b), sa.dot(sb))
     assert_eq(np.dot(a, b), sparse.dot(sa, sb))
 
+    if sys.version_info >= (3, 5):
+        assert_eq(eval("a @ b"), eval("sa @ sb"))
+        assert_eq(eval("sa @ sb"), sparse.dot(sa, sb))
```

Comment:
Cool.  If you wanted to test `__rmatmul__` then you might also add something like the following line:

    assert_eq(eval("a @ sb"), sparse.dot(a, sb))